backend/config/routes.rb

```diff
@@ -5,7 +5,10 @@
   namespace :api, defaults: { format: :json } do
     resources :users, only: %i[create show destroy]
     resource :session, only: %i[create destroy]
-    resources :restaurants, only: %i[index create show]
+    resources :restaurants, only: %i[index create show] do
+      resource :favorites, only: %i[create destroy]
+      resource :reviews, only: %i[create]
```

Comment:
Wait isn't this a duplicate from below? Also, it should probably be `resources :reviews` right?